exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,4 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+let totalOf = numberOfStudents + numberOfMentors
+console.log(`Percentage students: ${Math.round((numberOfStudents / totalOf) * 100)}%\nPercentage mentors: ${Math.round((numberOfMentors / totalOf) * 100)}%`);
```

Comment:
Good work Rafael, I suggest breaking your codes down into smaller pieces for better reading to prevent confusion. So it would be better if you declare more variables and the console.log it